middle_end/flambda2/parser/fexpr.ml

```diff
@@ -353,12 +353,14 @@ type binop =
   | Infix of infix_binop
   | String_or_bigstring_load of string_like_value * string_accessor_width
   | Bigarray_get_alignment of int
+  | Array_vector_load of Lambda.boxed_vector * array_kind * mutability
```

Comment:
this also needs changing if we decide to change how `128`-bits loads are represented in flambda2 primitives

Reply:
(Refactored)